uint8 SUCCESS     = 0
uint8 TIMEOUT     = 1
uint8 NO_SKELETON = 2
uint8 ERROR       = 3

uint8 result
